coordinator/core/src/main/kotlin/net/consensys/zkevm/ethereum/coordination/aggregation/AggregationTriggerCalculatorByBlobLimit.kt

```diff
@@ -0,0 +1,57 @@
+package net.consensys.zkevm.ethereum.coordination.aggregation
+
+import net.consensys.zkevm.domain.BlobCounters
+import net.consensys.zkevm.domain.BlobsToAggregate
+
+class AggregationTriggerCalculatorByBlobLimit(private val maxBlobsPerAggregation: UInt) :
+  SyncAggregationTriggerCalculator {
+
+  data class InFlightAggregation(
+    val blobsCount: UInt,
+    val blobsToAggregate: BlobsToAggregate,
+  )
+
+  private var inFlightAggregation: InFlightAggregation? = null
+
+  @Synchronized
+  override fun checkAggregationTrigger(blobCounters: BlobCounters): AggregationTrigger? {
+    val blobCount = (inFlightAggregation?.blobsCount ?: 0u) + 1u
+    return if (blobCount >= maxBlobsPerAggregation) {
```

Comment:
nit: can we refactor these two lines into a private helper boolean function as they are used in two places:
```
val blobCount = (inFlightAggregation?.blobsCount ?: 0u) + 1u
return blobCount >= maxBlobsPerAggregation
```